tools/archetype/binding/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,52 @@
+# <bindingName> Binding
+
+_Give some details about what this binding is meant for - a protocol, system, specific device._
+
+_If possible, provide some resources like pictures, a YouTube video, etc. to give an impression of what can be done with this binding._
+
+## Supported Things
+
+_Please describe the different supported things / devices within this section._
+_Which different types are supported, which models were tested etc.?_
+_Note that it is planned to generate some part of this based on the XML files within ```ESH-INF/thing``` of your binding._
+
+## Discovery
+
+_Describe the available auto-discovery features here. Mention for what it works and what needs to be kept in mind when using it._
+
+## Binding Configuration
+
+_If your binding requires or supports general configuration settings, please create a folder ```cfg``` and place the configuration file ```<bindingId>.cfg``` inside it. In this section, you should link to this file and provide some information about the options. The file could e.g. look like:_
+
+```
+# Configuration for the Philips Hue Binding
+#
+# Default secret key for the pairing of the Philips Hue Bridge.
+# It has to be between 10-40 (alphanumeric) characters 
+# This may be changed by the user for security reasons.
+secret=EclipseSmartHome
+```
+
+_Note that it is planned to generate some part of this based on the information that is available within ```ESH-INF/binding``` of your binding._
+
+_If your binding does not offer any generic configurations, you can remove this section completely._
+
+## Thing Configuration
+
+_Describe what is needed to manually configure a thing, either through the (Paper) UI or via a thing-file. This should be mainly about its mandatory and optional configuration parameters. A short example entry for a thing file can help!_
+
+_Note that it is planned to generate some part of this based on the XML files within ```ESH-INF/thing``` of your binding._
+
+## Channels
+
+_Here you should provide information about available channel types, what their meaning is and how they can be used._
+
+_Note that it is planned to generate some part of this based on the XML files within ```ESH-INF/thing``` of your binding._
+
+## Full Example
+
+_Provide a full usage example based on textual configuration files (*.things, *.items, *.sitemap)._
```

Comment:
I would suggest to leave this out for the moment and go with examples on the README only for the following reasons:
- Users will rather copy&paste examples into their own files; they would not want to have separate files for each bundle (and this is not a practice I would want to suggest)
- the focus for openHAB 2 is more on the auto discovery and UI config; DSLs are just "one option for the techies"
- the sitemaps are a kind of "legacy", the plan is to have a replacement in ESH some when: https://www.eclipse.org/forums/index.php/t/1064113/. So the sitemaps should not be put in the focus of the documentation.
- I want to start small and make it easy for people to provide a docu page (until this Sunday for the existing stuff!). We can later still see what should be enhanced. Docu generation will anyhow change a lot again and I am also thinking about adding resources in the bundle for a kind of online documentation.